package.json

```diff
@@ -50,14 +56,14 @@
     "eslint-plugin-promise": "4.0.1",
     "eslint-plugin-standard": "4.0.0",
     "eth-gas-reporter": "0.1.12",
+    "eth-sig-util": "^2.1.1",
     "ethereumjs-wallet": "0.6.2",
     "ethers": "4.0.13",
+    "pre-commit": "^1.2.2",
     "safe-buffer": "5.1.2",
     "solium": "^1.1.8",
-    "truffle": "4.1.14",
-    "web3": "^1.0.0-beta.30",
-    "pre-commit": "^1.2.2",
-    "eth-sig-util": "^2.1.1"
+    "truffle": "^5.0.10",
```

Comment:
@atvanguard  Did we also upgrade zeppelin?

Reply:
No, why is that required?

Reply:
New zeppelin has some functions we can use. @0xAshish has put some hacks around it. Now that we are upgrading things, upgrading zeppeling would make sense.

Reply:
Great! @0xAshish keep an eye out for any such opportunity to incorporate the features from zep.